packages/e2e/features/ui/components/authenticator/verify-user.feature

```diff
@@ -9,11 +9,12 @@ Feature: Verify User
     And I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.GetUserAttributeVerificationCode" } }' with fixture "verify-user-email"
 
   @angular @react @vue
-  Scenario: Redirect to "Verify" page
+  Scenario: Redirect to "Verify" page and verify custom header and footer text
     When I type my "email" with status "UNVERIFIED"
     And I type my password
     And I click the "Sign in" button
-    Then I see "Account recovery requires verified contact information"
+    Then I see "Enter Information:"
+    Then I see "Footer Information"
```

Comment:
Had to add this here, because I couldn't get the setup totp mfa custom slots feature to get a user in this unusual verify, but confirmed state.